modules/effects/arrow-effects-rx2-extensions/src/main/kotlin/arrow/effects/rx2/extensions/observablek.kt

```diff
@@ -180,7 +180,7 @@ fun ObservableK.Companion.monadErrorSwitch(): ObservableKMonadError = object : O
 }
 
 // TODO ObservableK does not yet have a Concurrent instance
-fun <A> ObservableK.Companion.fx(c: suspend AsyncContinuation<ForObservableK, *>.() -> A): ObservableK<A> =
+fun <A> ObservableK.Companion.fx(c: suspend AsyncContext<ForObservableK>.() -> A): ObservableK<A> =
```

Comment:
...I'd use MonadSyntax, AsyncSyntax, MonadThrowSyntax

Context is too loaded with Android terminology :P

Reply:
Fair enough :D